reference/artifacts/in-pipelines/index.md

```diff
@@ -7,28 +7,28 @@ sidebar:
 
 {% include toc %}
 
-> :warning: Much of the behavior defined here depends on looking up execution
-> history within Redis. As a result, deleting recent executions from Redis is
-> undefined behavior.
+> :warning: Much of the behavior described here depends on looking up execution
+> history in Redis. Deleting recent executions from Redis can cause
+> unexpected behavior.
 
-Once you have an idea of what an "artifact" is in Spinnaker, it's important to
-understand how they are used within pipelines. The idea is simple, artifacts
-arrive in a pipeline execution either from an external trigger or a stage, and
-are consumed by downstream stages based on pre-defined behavior. In order to
-explain how this works, we need to understand the concept of an "expected
-artifact".
+Now that you have an idea of [what an artifact is](../#about-spinnaker-artifacts) in Spinnaker, you need to
+understand how it's used within pipelines. An artifact arrives in a pipeline execution either from an external trigger (for example, code check-in) or by getting fetched by a stage. That artifact is then consumed by downstream stages based on pre-defined behavior. 
```

Comment:
I would use a different example than "code check-in" since while it's supported, it's a little outside of what Spinnaker specializes in. I would say instead "a docker image pushed to a registry".